manifests/m/MongoDB/Server/5.0.4/MongoDB.Server.installer.yaml

```
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: MongoDB.Server
+PackageVersion: 5.0.4
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://downloads.mongodb.org/windows/mongodb-windows-x86_64-5.0.4-signed.msi
+  InstallerSha256: F44191D6E1F187EE6FF5404F6095F8C8F621BDACA56E54099FD4D2266F4CB60F
+  InstallerSwitches:
+    Custom: ACCEPTEULA=1 ADDLOCAL=ServerService
+  UpgradeBehavior: install
+  ProductCode: "{7E5638F2-BF57-427A-A73C-EBC4E157BE3C}"
```

```suggestion
  ProductCode: "{AC8EEF39-F25C-43D2-A005-7031FE5B415C}"
```
wingetcreate or YamlCreate.ps1 should automatically do this for you.